utils/network.py

```diff
@@ -0,0 +1,255 @@
+import networkx as nx
+import pandas as pd
+import plotly.graph_objects as go
+
+
+def name_identifier(uuid: str, dfs: list[pd.DataFrame]) -> str:
+    """Returns the name of the entity given the entity's uuid
+
+    Args:
+        uuid: the uuid of the entity
+        List of dfs: dataframes that have a uuid column, and an 'name' or
+        'full_name' column
+    Return:
+        The entity's name
+    """
+    for df in dfs:
+        # first, check orgs df:
+        if "name" in df.columns:
+            name_in_org = df.loc[df["id"] == uuid]
+            if len(name_in_org) > 0:
+                return name_in_org.iloc[0]["name"]
+        # theoretically it must be in inds if not in orgs, but for the sample
+        # data this might not be the case
+
+        if "full_name" in df.columns:
+            name_in_ind = df.loc[df["id"] == uuid]
+            if len(name_in_ind) > 0:
+                return name_in_ind.iloc[0]["full_name"]
+    return None
+
+
+def combine_datasets_for_network_graph(dfs: list[pd.DataFrame]) -> pd.DataFrame:
+    """Combines the 3 dataframes into a single dataframe to create the graph
+
+    Given the inds, orgs, and transactions dataframes, the func first finds the
```

Comment:
work to shorten this doc string -- I think you're making this sound more complicated than it needs to be